test/CMakeLists.txt

```diff
@@ -121,6 +121,7 @@ foreach(_test Callbacks Degenerate ManufacturedSolution ComparisonWithBoost)
 endforeach()
 add_executable(ArborX_QueryTree.exe
   ${ARBORX_TEST_QUERY_TREE_SOURCES}
+  tstQueryTreeCallbacksRay.cpp
```

Comment:
File name is not good.

Also wondering if we should add a dedicated executable for it.
I suppose we could settle that later.  @Rombur @wjge what do you think?

Reply:
not sure of the criterion.